File: app/src/main/java/m13/retrofittest/main/database/AppDatabase.java
package m13.retrofittest.main.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import m13.retrofittest.main.repos.Repository;
import m13.retrofittest.main.repos.RepositoryDao;


@Database(entities = {Repository.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract RepositoryDao repositoryDao();
}

File: app/src/main/java/m13/retrofittest/main/database/AppDatabase.kt
package m13.retrofittest.main.database

import androidx.room.Database
import androidx.room.RoomDatabase

import m13.retrofittest.main.api.repos.Repository
import m13.retrofittest.main.api.repos.RepositoryDao

@Database(entities = [Repository::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): RepositoryDao
}